core/services/run_manager.go

```diff
@@ -224,19 +222,19 @@ func (jm *runManager) Create(
 	}
 
 	run.RunRequest = *runRequest
+	currentTaskRun := run.TaskRuns[0]
```

Comment:
Should this be `currentTaskRun := &run.TaskRuns[0]`? As it is, I think 'currentTaskRun` is a copy, which has no effect.

Reply:
You're right, removed.